example/README.md

```diff
@@ -0,0 +1,35 @@
+## Install and Run
+
+Download this example or clone [bottender-fb](https://github.com/bottenderjs/bottender-fb).
+
+```
+curl https://codeload.github.com/bottenderjs/bottender-fb/tar.gz/master | tar -xz --strip=2 bottender-fb-master/example
+cd example
+npm install
+npm run dev
+```
+
+You should fill your ACCESS_TOKEN, APP_SECRET, VERIFY_TOKEN, POST_ID in `server.js` before run your bot.
+
+> Note: `POST_ID` is something like `${page_id}_${story_fbid}` and we can get `story_fbid` in URL query string.  
```

Comment:
Is `in the form of` better than `something like`?

Reply:
fixed